src/backend/src/services/ws/index.js

```diff
@@ -145,6 +145,7 @@ export default class WebSocketManager {
 
       const won = client.lastCompletedRound === game.round;
       client.ws.send(JSON.stringify({ method: Events.END, endData: { won } }));
+      insertMatch(gameID, clientId, won, game.problemsPlayed);
```

Comment:
Missing await

Reply:
doesn't need await cause we don't need to wait for it